atomic_reactor/plugins/inject_yum_repos.py

```diff
@@ -174,14 +174,19 @@ def add_koji_repo(self):
         for platform in self.platforms:
             self.yum_repos[platform].append(yum_repo)
 
-    def _inject_into_repo_files(self, build_dir: BuildDir):
+    def _inject_repo_files(self, build_dir: BuildDir) -> None:
         """Inject repo files into a relative directory inside the build context"""
+        platform_repos = self.yum_repos[build_dir.platform]
+        if not platform_repos:
+            self.log.warning("no repos for %s, skipping this platform", build_dir.platform)
```

Comment:
can we change the message to :  

no repos for $platform, no repos to inject

Reply:
Saying "no repos" twice seemed a bit redundant, changed to

no repos to inject for $platform platform

let me know what you think

Reply:
right sounds good